validation/standard/init_test.go

```diff
@@ -0,0 +1,54 @@
+package standard
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigInitialization(t *testing.T) {
```

Comment:
What are your thoughts on running this as a test versus doing the check in the `init()` function and panicking if there is an error?

Reply:
Converting these from tests to checks in `init()` would allow us to catch any errors sooner (i.e. when the package is first initialized vs only when unit tests are run). However, negative test cases (ensure failure when an invalid value is detected) would still have to go into test files. Will leave as-is for now but open to refactoring later